apps/galaxy/src/modules/Orders/OpenOrders/DataTable.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from "react";
+import { chakra, Table, Tbody, Td, Th, Thead, Tr } from "@chakra-ui/react";
```

Comment:
If you check cc-multiverse/apps/galaxy/src/modules/auth/SignInForm or SignUpForm, these tags were imported to cc-multiverse/packages/ui-chakra/src folder first, and then import to this file. (The main aim is to reuse components from different apps.)

Reply:
I am not 100% sure if we need to do this in orders, but it feels more consistent.